source/guides/1.1/getting-started.md

```diff
@@ -319,7 +319,7 @@ To make our test pass, we need to edit our newly generated template file in `app
 </div>
 ```
 
-Save your changes and see your tests pass!
+Save your changes and see your tests pass! The output will also contain a skipped test. This is due to an autogenerated test in `spec/web/views/books/index_spec.rb`.
```

Comment:
I think this should be said explicitly, because it might confuse people.

Reply:
:+1: